core/Concurrency/Lock.php

```diff
@@ -1,11 +1,13 @@
 <?php
+
```

Comment:
Note: I've renamed various parameters and methods in this class, as when looking at it, without knowing what exactly the code is doing, the naming completely confused me. The old methods are still there for BC, as the class is used in some plugins.